<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="log" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="log">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node>
  
  <node name="image_proc" pkg="image_proc" type="image_proc" output="log">
    <remap from="/image_raw" to="/usb_cam/image_raw"/>
  </node>

  <node name="cameracalibrator" pkg="ptam" type="cameracalibrator" clear_params="true" output="screen">
    <remap from="image" to="/image_mono" />
    <remap from="pose" to="pose"/>
    <rosparam file="$(find ptam)/PtamFixParams.yaml"/>
  </node>

</launch>
